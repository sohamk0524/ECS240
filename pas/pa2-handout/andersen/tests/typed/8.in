c Test case with mix of all statements
p 155 240
t 1 30
t 2 30
t 3 30
t 4 30
t 5 30
t 6 29
t 7 29
t 8 29
t 9 29
t 10 29
t 11 28
t 12 28
t 13 28
t 14 28
t 15 28
t 16 27
t 17 27
t 18 27
t 19 27
t 20 27
t 21 26
t 22 26
t 23 26
t 24 26
t 25 26
t 26 25
t 27 25
t 28 25
t 29 25
t 30 25
t 31 24
t 32 24
t 33 24
t 34 24
t 35 24
t 36 23
t 37 23
t 38 23
t 39 23
t 40 23
t 41 22
t 42 22
t 43 22
t 44 22
t 45 22
t 46 21
t 47 21
t 48 21
t 49 21
t 50 21
t 51 20
t 52 20
t 53 20
t 54 20
t 55 20
t 56 19
t 57 19
t 58 19
t 59 19
t 60 19
t 61 18
t 62 18
t 63 18
t 64 18
t 65 18
t 66 17
t 67 17
t 68 17
t 69 17
t 70 17
t 71 16
t 72 16
t 73 16
t 74 16
t 75 16
t 76 15
t 77 15
t 78 15
t 79 15
t 80 15
t 81 14
t 82 14
t 83 14
t 84 14
t 85 14
t 86 13
t 87 13
t 88 13
t 89 13
t 90 13
t 91 12
t 92 12
t 93 12
t 94 12
t 95 12
t 96 11
t 97 11
t 98 11
t 99 11
t 100 11
t 101 10
t 102 10
t 103 10
t 104 10
t 105 10
t 106 9
t 107 9
t 108 9
t 109 9
t 110 9
t 111 8
t 112 8
t 113 8
t 114 8
t 115 8
t 116 7
t 117 7
t 118 7
t 119 7
t 120 7
t 121 6
t 122 6
t 123 6
t 124 6
t 125 6
t 126 5
t 127 5
t 128 5
t 129 5
t 130 5
t 131 4
t 132 4
t 133 4
t 134 4
t 135 4
t 136 3
t 137 3
t 138 3
t 139 3
t 140 3
t 141 2
t 142 2
t 143 2
t 144 2
t 145 2
t 146 1
t 147 1
t 148 1
t 149 1
t 150 1
t 151 0
t 152 0
t 153 0
t 154 0
t 155 0
s 0 1 -1 10
s 0 5 -1 9
s 0 3 -1 7
s 0 3 0 5
s 0 1 0 5
s 0 5 0 3
s 0 3 0 3
s 0 1 0 3
s 0 6 -1 15
s 0 10 -1 14
s 0 7 -1 15
s 1 5 0 10
s 1 1 1 1
s 1 3 1 3
s 1 5 1 5
s 0 6 1 3
s 0 11 -1 17
s 0 15 -1 16
s 0 12 -1 19
s 2 1 2 1
s 2 3 0 12
s 0 12 0 12
s 0 12 1 7
s 1 6 0 15
s 0 16 -1 23
s 0 20 -1 23
s 0 19 -1 25
s 2 10 2 6
s 1 11 1 12
s 1 15 3 1
s 1 15 2 10
s 0 16 2 10
s 0 21 -1 26
s 0 25 -1 30
s 0 23 -1 30
s 2 12 2 11
s 1 19 1 20
s 3 10 2 11
s 4 5 0 23
s 1 16 2 15
s 0 26 -1 35
s 0 30 -1 35
s 0 27 -1 33
s 3 11 2 16
s 0 30 5 3
s 2 16 4 6
s 2 19 4 7
s 0 27 3 11
s 0 31 -1 39
s 0 35 -1 39
s 0 34 -1 40
s 2 21 5 6
s 3 20 2 23
s 6 1 3 16
s 5 7 1 30
s 4 15 4 11
s 0 36 -1 43
s 0 40 -1 43
s 0 38 -1 43
s 1 34 2 27
s 6 10 0 38
s 4 19 2 26
s 0 38 1 34
s 6 10 5 12
s 0 41 -1 49
s 0 45 -1 46
s 0 43 -1 48
s 5 20 3 30
s 2 31 3 26
s 6 12 0 45
s 6 11 5 19
s 7 10 6 12
s 0 46 -1 52
s 0 50 -1 51
s 0 49 -1 54
s 7 11 3 31
s 6 20 7 15
s 9 3 0 49
s 3 31 0 50
s 9 5 2 40
s 0 51 -1 60
s 0 55 -1 60
s 0 54 -1 59
s 4 31 2 43
s 8 15 7 19
s 9 10 0 55
s 0 51 3 36
s 1 46 8 15
s 0 56 -1 61
s 0 60 -1 61
s 0 58 -1 64
s 9 15 5 35
s 2 46 2 50
s 9 15 9 12
s 10 6 11 3
s 10 6 0 60
s 0 61 -1 67
s 0 65 -1 70
s 0 64 -1 70
s 8 21 5 36
s 4 45 5 38
s 5 38 0 64
s 2 51 4 43
s 12 3 10 15
s 0 66 -1 72
s 0 70 -1 74
s 0 69 -1 71
s 13 3 12 10
s 6 38 10 20
s 2 60 13 1
s 7 31 7 31
s 10 16 2 56
s 0 71 -1 77
s 0 75 -1 77
s 0 74 -1 79
s 2 65 5 50
s 0 74 12 12
s 7 38 11 20
s 14 3 12 15
s 12 12 1 69
s 0 76 -1 82
s 0 80 -1 84
s 0 78 -1 84
s 4 60 12 20
s 6 49 0 80
s 1 75 15 3
s 12 19 10 30
s 14 6 7 45
s 0 81 -1 87
s 0 85 -1 86
s 0 82 -1 86
s 1 78 13 20
s 11 26 11 30
s 8 45 5 56
s 10 31 4 61
s 15 6 0 85
s 0 86 -1 91
s 0 90 -1 93
s 0 88 -1 95
s 16 7 4 70
s 6 60 9 45
s 6 56 10 40
s 7 54 15 11
s 15 11 9 45
s 0 91 -1 100
s 0 95 -1 100
s 0 92 -1 96
s 18 1 7 60
s 17 7 6 64
s 5 70 16 15
s 12 31 14 21
s 5 66 3 80
s 0 96 -1 103
s 0 100 -1 102
s 0 98 -1 104
s 15 21 17 11
s 0 98 19 3
s 16 16 0 100
s 15 25 3 85
s 4 78 10 49
s 0 101 -1 106
s 0 105 -1 107
s 0 104 -1 108
s 4 81 3 86
s 7 69 16 23
s 0 104 12 43
s 1 98 0 105
s 1 100 16 23
s 0 106 -1 115
s 0 110 -1 112
s 0 109 -1 111
s 16 27 13 43
s 19 15 0 106
s 5 82 16 26
s 14 38 7 75
s 6 80 17 21
s 0 111 -1 117
s 0 115 -1 116
s 0 113 -1 117
s 18 23 21 7
s 9 69 11 56
s 2 105 6 81
s 4 91 14 43
s 9 66 0 113
s 0 116 -1 125
s 0 120 -1 123
s 0 117 -1 124
s 2 110 14 50
s 20 19 8 76
s 19 21 22 6
s 11 65 19 21
s 17 34 5 91
s 0 121 -1 130
s 0 125 -1 126
s 0 123 -1 127
s 4 104 8 85
s 1 116 14 51
s 17 36 11 66
s 23 7 2 111
s 19 26 2 111
s 0 126 -1 133
s 0 130 -1 131
s 0 128 -1 131
s 1 125 17 43
s 4 106 8 86
s 16 49 24 10
s 1 121 18 38
s 5 104 2 120
s 0 131 -1 139
s 0 135 -1 140
s 0 133 -1 137
s 24 11 22 25
s 26 5 9 86
s 3 117 24 11
s 5 110 9 86
s 21 30 1 126
s 0 136 -1 142
s 0 140 -1 141
s 0 138 -1 141
s 17 55 5 113
s 20 36 13 74
s 3 121 21 34
s 5 113 21 35
s 13 75 24 19
s 0 141 -1 146
s 0 145 -1 150
s 0 142 -1 148
s 13 78 0 142
s 14 75 16 61
s 2 133 15 70
s 14 74 19 50
s 2 133 2 131
s 0 146 -1 153
s 0 150 -1 155
s 0 149 -1 151
s 14 78 0 150
s 22 40 1 145
s 25 25 16 70
s 23 34 0 150
s 6 120 12 86
