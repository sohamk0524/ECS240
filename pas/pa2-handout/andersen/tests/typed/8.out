pt 100 102
pt 101 106
pt 10 14
pt 102 107
pt 104 107
pt 104 108
pt 105 107
pt 106 115
pt 107 115
pt 109 111
pt 1 10
pt 110 112
pt 111 117
pt 11 17
pt 113 117
pt 115 116
pt 116 125
pt 117 124
pt 120 123
pt 121 130
pt 12 16
pt 12 19
pt 123 127
pt 125 126
pt 126 133
pt 128 131
pt 130 131
pt 130 133
pt 131 139
pt 133 137
pt 135 140
pt 136 142
pt 138 141
pt 140 141
pt 141 146
pt 14 16
pt 14 19
pt 142 148
pt 145 150
pt 146 153
pt 149 151
pt 150 155
pt 15 16
pt 15 19
pt 16 23
pt 16 25
pt 1 7
pt 17 23
pt 17 25
pt 1 9
pt 19 23
pt 19 25
pt 20 23
pt 21 26
pt 23 30
pt 25 30
pt 26 35
pt 27 33
pt 27 35
pt 30 35
pt 31 39
pt 34 40
pt 35 39
pt 36 43
pt 3 7
pt 38 43
pt 3 9
pt 39 43
pt 40 43
pt 41 49
pt 43 46
pt 43 48
pt 45 46
pt 46 51
pt 46 52
pt 46 54
pt 48 51
pt 48 54
pt 49 54
pt 50 51
pt 51 59
pt 51 60
pt 52 59
pt 52 60
pt 54 59
pt 54 60
pt 55 60
pt 56 61
pt 5 7
pt 58 64
pt 5 9
pt 59 61
pt 60 61
pt 6 14
pt 6 15
pt 61 67
pt 61 70
pt 64 70
pt 65 70
pt 66 72
pt 69 71
pt 70 74
pt 7 14
pt 7 15
pt 71 77
pt 74 77
pt 74 79
pt 75 77
pt 76 82
pt 77 84
pt 78 84
pt 79 84
pt 80 84
pt 81 87
pt 82 86
pt 84 86
pt 85 86
pt 86 91
pt 88 95
pt 90 93
pt 91 100
pt 9 14
pt 92 96
pt 95 100
pt 96 103
pt 98 102
pt 98 104
